docker-entrypoint

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash -e
```

Comment:
new rails apps generate this file in `/bin`

Reply:
Updated! [69fe7ee](https://github.com/cedarcode/mi_carrera/pull/337/commits/69fe7eede05ac02819727f7216efd4871a3dc3e7)